### PR TITLE
Make VizTypeControl use metadata from plugin

### PR DESCRIPTION
**Pull request by kristw**
- Read available visualization types from `ChartMetadataRegistry` (i.e. the plugin system) instead of `visTypes` (which is the file that defines control panel and hard-code vis names).
- Same procedure with above for `addSlice` page.
- Extract `css` for `VizTypeControl` component into its own file. 
- Remove duplicate large thumbnail files
- Update look. (slightly larger font, change border to only the thumbnail and border color, slightly larger search box and reduce placeholder to `Search` instead of `Search/Filter`)

**Before**

image

**After**

image

@williaster @conglei @xtinec @graceguo-supercat @michellethomas  @elibrumbaugh

### CODE REVIEW

**Comment by mistercrunch** [comment]
Side mission would be for the control to offer a quick-toggle for the popular chart types that would not require opening the modal. 

Here's a horrible sketch of what I'm thinking, where the last button `...` opens the modal :
screen shot 2018-10-30 at 10 14 07 pm

@elibrumbaugh ^

**Comment by kristw** [comment]
Would be an interesting addition. I created an issue #6253 for tracking. 
Will leave this PR as-is as it seems a decent size already.

File: superset/assets/src/explore/components/controls/VizTypeControl.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.viztype-selector-container.selected img {
+  border: 1px solid #333;
+}
+
+.viztype-selector-container img {
+  border: 1px solid #ddd;
+  border-radius: 4px;
+  transition: border-color .2s;
```
Comment (williaster) [inline superset/assets/src/explore/components/controls/VizTypeControl.css]: ✨ 💁‍♀️

**Review by williaster — APPROVED** [review]
LGTM! solid UI improvement with constant size thumbnails, too! 🏆

**Comment by kristw** [comment]
ready to merge

**Comment by andreaslordos** [comment]
If I wrote a new plugin in my local clone of the of [superset-ui-plugins repository](https://github.com/apache-superset/superset-ui-plugins), how do I include the metadata (specified in index.js of the plugin folder) in ChartMetadataRegistry? 

Is it automatically included once configured & registered in MainPreset.js?

**Pull request merged**